dataworks-aws-data-egress.tf

```diff
@@ -0,0 +1,81 @@
+locals {
+  dataworks_aws_data_egress_pipeline_name = "dataworks_aws_data_egress"
```

Comment:
you might want this to be the shorter form:
```suggestion
  dataworks_aws_data_egress_pipeline_name = "data_egress"
```

Reply:
done